server_admin/topics/clients/client-policies.adoc

```diff
@@ -0,0 +1,159 @@
+=== Client Policies
+
+To make it easy to secure client applications, it is beneficial to realize the following points in a unified way.
+
+* Setting policies on what configuration a client can have
+* Validation of client configurations
+* Conformance to a required security standards and profiles such as FAPI
+
+To realize these points in a unified way, _Client Policies_ concept is introduced.
+
+==== Use-cases
+
+Client Policies realize the following points mentioned as follows.
+
+Setting policies on what configuration a client can have::
+    Configuration settings on the client can be enforced by client policies during client creation/update, but also during OpenID Connect requests to {project_name} server, which are related to particular client.
+    The {project_name} supports similar thing also through the Client Registration Policies described in the link:{adapterguide_link}#_client_registration_policies[{adapterguide_name}].
+    However, Client Registration Policies can only cover OIDC Dynamic Client Registration. Client Policies cover not only what Client Registration Policies can do, but other client
+    registration and configuration ways. In the future, Client Registration will be replaced by Client Policies in the future.
+
+Validation of client configurations::
+    The {project_name} supports validation whether the client follows settings like Proof Key for Code Exchange,
+    Request Object Signing Algorithm, Holder-of-Key Token, etc on some endpoints like Authorization Endpoint, Token Endpoint, etc.
+    These can be specified by each setting item (on Admin Console, switch, pulldown menu, etc). To make the client application secure, administrator needs to set
+    lot of such settings in appropriate way, which makes it difficult for the administrator to secure their client application.
+    Client Policies can do these validation of client configurations mentioned just above and they can also be used to auto-configure some client configuration switches to meet
+    the advanced security requirements. In the future, individual client configuration settings may be replaced by Client Policies directly performing required validations.
+
+Conformance to a required security standards and profiles such as FAPI::
+    The _Global client profiles_ are client profiles pre-configured in {project_name} by default. They are pre-configured to be compliant with standard security profiles like link:{adapterguide_link}#_fapi-support[FAPI],
+    which makes it easy for the administrator to secure their client application to be compliant with the particular security profile. At this moment, {project_name} has global
+    profiles for the support of FAPI 1 specification. Administrator will just need to configure the client policies to specify which clients should
+    be compliant with the FAPI. Administrator can configure his own client profiles and client policies, so that {project_name} clients can be easily made compliant with various other
```

Comment:
The administrator

Concerning "his" what if the administrator is a woman.  Remove "his own"